engine/table/src/main/java/io/deephaven/engine/table/impl/lang/QueryLanguageFunctionUtils.java

```diff
@@ -19274,6 +19274,8 @@ public static byte byteCast(short a) {
     }
 
```

Comment:
This is a comment relevant to this whole file, not this specific line. Can we (either in this PR, or in a followup PR) bring things up to date so that `QueryLanguageFunctionGenerator.java` actually generates this file like it is supposed to?

I tried to run the `main` inside `QueryLanguageFunctionGenerator.java` but I got a runtime error having to do with demand-loading the logger. Discussion on internal Slack at https://illumon.slack.com/archives/G01K6UHJ00K/p1671933281407829

Anyway I think that's how this stuff was *intended* to work.

Reply:
Yeah, that is how it works/used to work. I couldn't get it running either.

#3262 (which I just realized duplicates #2184)